ui/lobby/src/lobby.ts

```diff
@@ -19,12 +21,12 @@ export function initModule(opts: LobbyOpts) {
     { id: '30+0', lim: 30, inc: 0, perf: 'Classical' },
     { id: '30+20', lim: 30, inc: 20, perf: 'Classical' },
   ];
-  opts.trans = site.trans(opts.i18n);
+  opts.trans = trans(opts.i18n);
 
-  site.socket = new site.StrongSocket('/lobby/socket/v5', false, {
+  site.socket = new StrongSocket('/lobby/socket/v5', false, {
     receive: (t: string, d: any) => lobbyCtrl.socket.receive(t, d),
     events: {
-      n(_: string, msg: { d: number; r: number }) {
+      n(_: string, msg: any) {
```

Comment:
I don't understand how that still compiles:

```ts
lobbyCtrl.spreadPlayersNumber(msg.d)
```

now that `msg.d: any` while spreadPlayersNumber expects a `number`